airflow/traces/tracer.py

```diff
@@ -54,19 +55,15 @@ def wrapper(*args, **kwargs):
         else:
             component = module_name
```

Comment:
Nit: since we are in here, lets move this name extraction loggic outside of `wrapper`  so it's not recomputed every time we call the wrapper